src/simulator_adapter.py

```diff
@@ -50,6 +50,16 @@ def sim_pose_callback(pose_msg):
 
         tf2_broadcaster.sendTransform(transform_msg)
 
+    if publish_ground_truth_camera_localization:
+        camera_pose_msg = PoseWithCovarianceStamped()
+        camera_pose_msg.header.stamp = pose_msg.header.stamp
+        camera_pose_msg.header.frame_id = 'map'
+        camera_pose_msg.pose.pose.position = pose_msg.pose.position
```

Comment:
Should we just copy the parts of the pose out that the camera would actually report?